randovania/games/patchers/claris_patcher_file.py

```diff
@@ -402,6 +402,10 @@ def create_patcher_file(description: LayoutDescription,
     result = {}
     _add_header_data_to_result(description, result)
 
+    result["game_id"] = "G2ME0R"
```

Comment:
This breaks non-NTSC, as `G2ME` is the NTSC id.

Reply:
good catch, will change in the morning so it only modifies the publisher ID